src/main/paradox/docs/releases/index.md

```diff
@@ -1,5 +1,7 @@
 @@@ index
 
+* [v1.2 To v1.3 Migration](v1.2-to-v1.3-migration.md)
```

Comment:
Why is this the first item and not `v1.3 Release notes`?

At the bottom be have `v1.1 Release Notes` and after `v1.0 To v1.1 Migration`

Reply:
Done.